### PR TITLE
[docs] Improve instructions for using Ninja with Xcode & move them to GettingStarted.md

### LINKED ISSUE #61907: Expand Erik's documentation on integrating Ninja with Xcode into a proper tutorial

### PR DESCRIPTION
**Pull request by AnthonyLatsis**

### CODE REVIEW

**Review by LucianoPAlmeida — APPROVED** [review]
Looks Great!

File: docs/HowToGuides/GettingStarted.md [hunk 1]
```diff
@@ -368,6 +351,82 @@ select the following schemes:
   run.
 <!-- TODO: Insert SourceKit/stdlib specific instructions? -->
 
+### Using Ninja with Xcode
+
+Although it's possible to build the toolchain entirely with Xcode via `--xcode`,
+a more efficient and robust option is to integrate a Ninja build with Xcode.
+This is also convenient in that you can navigate, build, run, edit and debug in
```
Comment (xedin) [inline docs/HowToGuides/GettingStarted.md:358]: Nit: missing serial comma before “and debug”

**Review by xedin — APPROVED** [review]
Awesome, thank you!

File: docs/HowToGuides/GettingStarted.md (hunk 1 above)
Comment (AnthonyLatsis) [reply docs/HowToGuides/GettingStarted.md:358]: A serial comma in this non-ambiguous case is regarded a preference as far as I know. Not to say I have a strong preference—will do.

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test macOS

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test macOS

**Comment by AnthonyLatsis** [comment]
@swift-ci please clean smoke test macOS

**Pull request merged**